docs/source/introduction/tutorial/composing_pipelines/automated_creation.rst

```diff
@@ -15,10 +15,11 @@ Automated way
     See the :doc:`API documentation </api/api>` for descriptions of these and others parameters.
 
 .. code:: python
-
+   import fedot
    # build model
    auto_model = Fedot(
-      problem='classification', timeout=timeout, preset='fast_train', 
+      problem=fedot.api.main.TaskTypesEnum.classification.value, timeout=timeout, 
```

Comment:
It's better to import TaskTypesEnum and PresetTypesEnum somewhere above and pass only`TaskTypesEnum.classification` or `PresetTypesEnum.fast_train` into FEDOT constructor.

Reply:
Also, see below for my proposal about support of both string-based and Enum-based params (since the external Enum-based params are quite rare for Python ML libraries)

Reply:
Please can you suggest any code to me that I can replace with my implementaion. I am littlebit new around this repo. Thanks

Reply:
in which file the fedot constructor is located? I want to make the suggested corrections

Reply:
>which file the fedot constructor is located

See "class Fedot" in https://github.com/aimclub/FEDOT/blob/master/fedot/api/main.py